indico/core/signals/event/registration.py

```diff
@@ -0,0 +1,24 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from blinker import Namespace
+
+_signals = Namespace()
+
+
+get_regform_field_types = _signals.signal('get-regform-fields', """
+Expected to return `FieldType` subclasses.
```

Comment:
Maybe mention that you can't just use the signal to add a new field type (since the angular code would probably need to know about new field types as well)